src/components/select/select.scss

```diff
@@ -11,6 +11,12 @@
     display: none;
```

Comment:
I like that all the changes to limel-select are contained in a single commit. I think all the changes to limel-picker also belong in a single commit, and the changes to limel-portal, limel-menu, and limel-menu-surface belong in a single commit (which should come before the others).

I'll move stuff around a little on a separate branch, and you can see if you like it better my way or not 🙂

Reply:
I agree 🙂

Reply:
⚡